idutils/normalizers.py

```diff
@@ -187,8 +187,8 @@ def normalize_pid(val, scheme):
     "ascl": "{scheme}://ascl.net/{pid}",
     "orcid": "{scheme}://orcid.org/{pid}",
     "pmid": "{scheme}://pubmed.ncbi.nlm.nih.gov/{pid}",
+    "pmcid": "{scheme}://pmc.ncbi.nlm.nih.gov/articles/{pid}",
```

Comment:
So the "canonical" url format has a trailing slash at the end per https://pmc.ncbi.nlm.nih.gov/about/userguide/#urlformat :

```suggestion
    "pmcid": "{scheme}://pmc.ncbi.nlm.nih.gov/articles/{pid}/",
```

On 1 hand) without a slash is fine as it will redirect to the one with, on 2 hand) if we provide it, it skips a redirect and uses the "canonical"/"preferred" one, and on 3 hand) I don't know if trailing slash plays well with idutils logic.

So either way is fine, my slight preference would be for the trailing slash form.

Reply:
I switched both pmcid and pmc to have the trailing slash, and they work fine when deployed to RDM.